src/genie/libs/parser/iosxe/show_ip_nbar_version.py

```diff
@@ -0,0 +1,118 @@
+"""show_ip_nbar_version.py
+    supported commands:
+        * show ip nbar version
+
+"""
+
+# Python
+import re
+
+# Metaparser
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import (Schema,
+                                                Any,
+                                                Or,
+                                                Optional)
+
+# parser utils
+from genie.libs.parser.utils.common import Common
+
+# ==============================
+# Schema for:
+#    * 'show ip nbar version'
+# ==============================
+class ShowIpNbarVersionSchema(MetaParser):
+    """ Schema for the commands:
+            *  show ip nbar version
+    """
+
+    schema = {
+        'file': str,
+        'name': str,
+        'publisher': str,
+        'nbar_minimum_backward_compatible_version': str,
+        'creation_time': str,
+        'nbar_engine_version': str,
+        'nbar_software_version': str,
+        'state': str,
+        'version': str
+    }
+
+# ==============================
+# Parser for:
+#    * 'show ip nbar version'
+# ==============================
+class ShowIpNbarVersion(ShowIpNbarVersionSchema):
+    '''
+    Parser for:
+    show ip nbar version
+    '''
+
+    cli_command = ['show ip nbar version']
+
+    #* could add extra parameters if needed
+    def cli(self, output = None):
+        if output is None:
+            out = self.device.execute(self.cli_command)
+
+        else:
+            out = output
+
+        # initial variables
+        parsed_dict = {}
+
+        p1 = re.compile('^NBAR software version:\s+(?P<nbar_software_version>.+)')
```

Comment:
like our parser code. please have sample output as comment to verify the regex like below.
```suggestion
        # NBAR software version:  34
        p1 = re.compile('^NBAR software version:\s+(?P<nbar_software_version>.+)')
```

Reply:
Fixed!